cairo1-run/src/cairo_run.rs

```diff
@@ -1447,8 +1447,8 @@ fn serialize_output_inner<'a>(
                 output_string.push(':');
                 // Serialize the value
                 // We create a peekable array here in order to use the serialize_output_inner as the value could be a span
-                let value_vec = vec![value.clone()];
-                let mut value_iter = value_vec.iter().peekable();
+                let value_arr = [value.clone()];
```

Comment:
While `v1.0.2` is a branch of [main](https://github.com/lambdaclass/cairo-vm/blob/7c1b9d92b703226e5fcaac30b36bc7fbd17369e4/cairo1-run/src/cairo_run.rs#L1476) I would leave it as it is in main to decrease diffs between both branches:

```suggestion
                let value_vec = [value.clone()];
```

Reply:
Done in [`0e59237`](https://github.com/lambdaclass/cairo-vm/pull/2112/commits/0e59237578e57647b5f6e8b5a7421faa2b171f41)!